server/lorax_server/models/flash_causal_lm.py

```diff
@@ -731,7 +732,11 @@ def warmup(self, batch: FlashCausalLMBatch):
                 self.dtype,
                 self.device,
             )
-            _, batch = self.generate_token(batch)
+
+            with tqdm(total=max_new_tokens, desc="Warmup to max_total_tokens") as pbar:
```

Comment:
Nit: could be nice to put the actual value of `max_total_tokens` in as well.

Reply:
Good point, though will require a bit more plumbing. We do show the max new tokens from the progress bar counter, so will leave as is for now.